app/src/main/java/com/github/houseorganizer/houseorganizer/task/TaskAssigneeAdapter.java

```diff
@@ -44,7 +43,9 @@ public void onBindViewHolder(@NonNull BiViewHolder<TextView, ImageButton> holder
         boolean isAssigned = isAmongAssignees(assigneeEmail);
         assigneeEmailTextView.setText(assigneeEmail);
 
-        imgButton.setBackgroundResource(isAssigned ? R.drawable.remove_person : R.drawable.add_person);
+        int rscId = isAssigned ? R.drawable.remove_person : R.drawable.add_person;
+        imgButton.setBackgroundResource(rscId);
+        imgButton.setTag(rscId);
```

Comment:
it might look cleaner if you chain these calls like imgButton.setBackgroundResource(rscId).setTag(rscId);

Reply:
Unfortunately setBackgroundResource has a void return :(